src/Allocs.jl

```diff
@@ -43,7 +43,7 @@ function pprof(alloc_profile::Profile.Allocs.AllocResults = Profile.Allocs.fetch
                # Allocs-specific arguments:
                frame_for_type::Bool = true,
             )
-    period = UInt64(0x1)
+    period = sum(alloc.size for alloc in alloc_profile.allocs, init=0)
```

Comment:
🤔 this seems weird.. Isn't this supposed to basically represent how much _whatever_ does one sample represent?

So for a CPU profile, this would be like how much time in between samples? So for an allocation profile, it should be 1 allocation per sample, right?

Reply:
Hm yeah I was misunderstanding this.

Looking at the proto, it says "The number of events between sampled occurrences": https://github.com/google/pprof/blob/131d412537eacd9973045c73835c3ac6ed696765/proto/profile.proto#L85-L86

So I guess it's basically supposed to be the sample interval? E.g. "every 1000th alloc" would be 1000?